brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3486 Fix: Slider - Resolved an issue of block encounter an issue and can not be previewed in Customizing > Widgets of Astra.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3486

### Screenshots

- https://d.pr/v/siFHJ0

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate Astra Pro and Astra theme
- Go to Customizer > Widgets > Select any footer widget
- Drag and drop Slider
- Make sure it does not throw an Error encounter and can not be previewed.

### Checklist:
